sub-logs/anuj.json

```diff
@@ -0,0 +1,11 @@
+{
+  "repo": "https://github.com/qanuj/register-good",
+  "owner": {
+    "username": "qanuj",
+    "email": "anuj@anuj.co.in",
+    "twitter": "qanuj"
+  },
+  "record": {
+    "URL": "https://anujpandey.com"
```

Comment:
Sorry, we don't support URL records. Only A and CNAME. Please use a redirect service such as https://redirect.pizza